core/trino-main/src/main/java/io/trino/sql/planner/sanity/ValidateDependenciesChecker.java

```diff
@@ -657,6 +658,12 @@ public Void visitTableExecute(TableExecuteNode node, Set<Symbol> boundSymbols)
             return null;
         }
 
+        @Override
+        public Void visitTableExecuteCoordinatorOnly(NonReadingTableExecuteNode node, Set<Symbol> context)
+        {
+            return null;
```

Comment:
why not ` PlanNode source = node.getSource();
            source.accept(this, boundSymbols); // visit child` ?

Reply:
cause there is no source?